app/views/organizations/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= c 'main', style: :lightestGrayBackground do %>
+  <%= c 'page_header', styles: [:wide, :inheritBackgroundColor, :xlargePaddingTop, :spaceBetween] do %>
+    <%= c 'heading' do %>
+      <%= t ".heading", first_name: current_user.first_name %>
+    <% end %>
+  <% end %>
+
+
+  <%= c 'section', styles: [:wide, :largeBottomMargin], class: 'OrganizationsList' do %>
+    <ul>
+      <% @organizations.each do |organization| %>
+        <li><%= link_to organization.name, organization_dashboard_path(@organization) %><li>
+      <% end %>
+    <ul>
+  <% end %>
+<% end %>
```

Comment:
I'm also not satisfied with how this looks. I proposed some changes that I think are an improvement of this in https://github.com/tactilenews/100eyes/pull/1969

It would look like this.

![Screenshot_20240812_112111](https://github.com/user-attachments/assets/248d85b5-4401-439e-9cda-6f3a068df299)
![Screenshot_20240812_112138](https://github.com/user-attachments/assets/cb206785-4a93-4e67-bef0-c101baeaf2e9)

It's an auto submit form that sets the organization for the admin/user with multiple organizations.

It does not include the navbar since we don't want to give a way to navigate around the site if we don't know which organization to set.

Reply:
thnx, just integrated most of your proposal with https://github.com/tactilenews/100eyes/pull/1953/commits/703942b3d4e44e96193f625ce36845ad353b85b1

is there any reason that you didn't use plain old links so that we have the form and route overhead?

Reply:
I'm not sure what you mean by:

> so we have the form and route overhead

Plain old links could have some benefits over a select. They are a little less flexible when you have many options, but in this use case, it's hard to say what the likelihood is that we will have 20 organization, much less that some user will be part of all 20.

When I looked at this bit of code, it wasn't obvious to me why the extra link was being created. I did not attempt to style the links to remove the native browsers bullet points, which probably would have hidden this extra empty link.